src/latexify/codegen/algorithmic_codegen.py

```diff
@@ -3,6 +3,8 @@
 from __future__ import annotations
 
 import ast
+import contextlib
+from typing import Generator
```

Comment:
Use `collections.abc` rather than `typing` for generic types.

```suggestion
from collections.abc import Generator
```

Reply:
just curious as to why not use typing